src/components/tabs/index.scss

```diff
@@ -0,0 +1,31 @@
+@import "src/components/style/base.scss";
+
+.tabs {
+    &__nav {
+        background: var(--tabsHeaderBackgroundColor);
+        border-bottom: 1px solid var(--dividerColor);
+        margin: 0;
```

Comment:
do not add extra props. By default `div` always has `margin: 0; padding: 0`
Plz update all your SCSS files

Reply:
Done

Reply:
I don't see changes